tests/components/habitica/test_integration.py

```diff
@@ -0,0 +1,142 @@
+"""Test the habitica module."""
+import pytest
+
+from homeassistant.components.habitica.const import (
+    ATTR_ARGS,
+    ATTR_DATA,
+    ATTR_PATH,
+    DEFAULT_URL,
+    DOMAIN,
+    EVENT_API_CALL_SUCCESS,
+    SERVICE_API_CALL,
+)
+from homeassistant.components.habitica.sensor import TASKS_TYPES
+from homeassistant.const import ATTR_NAME
+from homeassistant.core import callback
+
+from tests.common import MockConfigEntry
+
+TEST_API_CALL_ARGS = {"text": "Use API from Home Assistant", "type": "todo"}
+TEST_USER_NAME = "test_user"
+
+
+@pytest.fixture
+def capture_api_call_success(hass):
+    """Capture api_call events."""
+    events = []
+
+    @callback
+    def async_capture(event):
+        events.append(event.data)
+
+    hass.bus.async_listen(EVENT_API_CALL_SUCCESS, async_capture)
```

Comment:
We can use this test helper instead:
https://github.com/home-assistant/core/blob/22c8afe637a98a82c79f2b636c81473663e7bbc1/tests/common.py#L1106